contracts/Certificate.sol

```diff
@@ -119,7 +120,7 @@ contract Certificate is ERC1155PresetPausableNonTransferrable {
     uint256,
     bytes memory
   ) public pure override {
-    revert ForbiddenFunctionCall(); // todo is this really what we want?
+    revert MintDisabled();
```

Comment:
I would rather just change how this works. We should:
- stop inheriting `ERC1155PresetMinterPauserUpgradeable` in `ERC1155PresetPausableNonTransferrable`
- change the `mint` to allow minting `n` removals for one supplier
- change `mintBatch` to allow minting `n` removals for `m` suppliers
- perhaps this needs to be further extended one more dimension for projects?

Reply:
do you think we should stop inheriting burning in `ERC1155PresetPausableNonTransferrable` too? (`ERC1155PresetMinterPauserUpgradeable` inherits burning, so when we stop inheriting `ERC1155PresetMinterPauserUpgradeable` we can decide at a granular level what we want to pull in.  maybe just pausable and no burning or minting since we have no need for burning on certificates?)

Reply:
Honestly, might even be worth not inheriting any of the presets since none of them use custom errors. Up to you!